Workspaces/MiniWorldWorkspace/Shaders/Grid.shader

```diff
@@ -33,6 +34,7 @@
 				uniform float2 _GridSpacing;
 				uniform float4 _GridCenter;
 				uniform float _GridScale;
+				uniform float _GridFade;
```

Comment:
There are some other fixes I'd like to make, as well. You can get some weird issues when  you zoom out really far, pan, and zoom back in due to floating point precision issues. This one was very noticeable whenever you resize the workspace, which is why it's addressed in this PR.